modules/installation-nutanix-infrastructure.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/preparing-to-install-nutanix.adoc
+
+:_content-type: CONCEPT
+[id="installation-nutanix-infrastructure_{context}"]
+= Nutanix version requirements
+
+You must install the {product-title} cluster on Nutanix AOS version <to do> (6.0.1 or later?) that meets the following requirements.
+
+
+.Version requirements for Nutanix virtual environments
+[cols=2, options="header"]
+|===
+|Virtual environment product |Required version
+|VM hardware version | TBD
+|Nutanix AHV | TBD
+|Prism Central | TBD
+|===
```

Comment:
Are there any other components we need to account for? Are AOS, AHV, and Prism Central versioned separately? If they are upgraded separately, we might have to explicitly call out what combinations we are supporting. Or, when I upgrade AOS, do other components automatically get upgraded?

Reply:
We just need Nutanix **AOS**, and Prism Central. They are upgraded separately but the plan is to limit scope by supporting the latest Nutanix release unless it is initial release in train (eg x.y.0) at OCP release time.

Reply:
> supporting the latest Nutanix release

Please provide the release I should be adding, keeping in mind that we are GA in August. Thank you.

Reply:
@mjpytlak Nutanix AOS 5.20.4 and 6.1.1 with Prism Central 2022.4

Reply:
Added to doc.Thank you.